frontend/src/roster/pages/CharacterCreatePage.tsx

```diff
@@ -0,0 +1,192 @@
+import { useState } from 'react';
+import { Navigate } from 'react-router-dom';
+import { useAccount } from '@/store/hooks';
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '@/components/ui/card';
+import { Button } from '@/components/ui/button';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/ui/tabs';
+import { AlertCircle } from 'lucide-react';
+
+export function CharacterCreatePage() {
+  const account = useAccount();
+  const [activeTab, setActiveTab] = useState('name');
+
+  if (!account) {
+    return <Navigate to="/login" replace />;
+  }
+
+  if (!account.can_create_characters) {
+    return (
+      <div className="container max-w-4xl py-8">
+        <Card className="border-destructive">
+          <CardHeader>
+            <CardTitle className="flex items-center gap-2 text-destructive">
+              <AlertCircle className="h-5 w-5" />
+              Cannot Create Character
+            </CardTitle>
+            <CardDescription>
+              {!account.email_verified
+                ? 'You must verify your email address before creating characters.'
+                : 'You are not currently able to create new characters.'}
+            </CardDescription>
+          </CardHeader>
+        </Card>
+      </div>
+    );
+  }
+
+  return (
+    <div className="container max-w-6xl py-8">
+      <div className="mb-8">
+        <h1 className="text-3xl font-bold">Create Character</h1>
+        <p className="text-muted-foreground">
+          Fill out each section to create your character application.
+        </p>
+      </div>
+
+      <Tabs value={activeTab} onValueChange={setActiveTab} className="space-y-6">
+        <TabsList className="flex h-auto flex-wrap gap-1">
+          <TabsTrigger value="name">Name</TabsTrigger>
+          <TabsTrigger value="species">Species</TabsTrigger>
+          <TabsTrigger value="homeland">Homeland</TabsTrigger>
+          <TabsTrigger value="stats">Stats</TabsTrigger>
+          <TabsTrigger value="path">Path</TabsTrigger>
+          <TabsTrigger value="skills">Skills</TabsTrigger>
+          <TabsTrigger value="advantages">Traits</TabsTrigger>
+          <TabsTrigger value="personality">Personality</TabsTrigger>
+          <TabsTrigger value="description">Description</TabsTrigger>
+          <TabsTrigger value="background">Background</TabsTrigger>
+          <TabsTrigger value="relationships">Relationships</TabsTrigger>
+        </TabsList>
+
+        <TabsContent value="name">
+          <Card>
+            <CardHeader>
+              <CardTitle>Character Name</CardTitle>
+              <CardDescription>Choose your character's first name.</CardDescription>
+            </CardHeader>
+            <CardContent>
+              <p className="text-muted-foreground">TODO: Name input</p>
+            </CardContent>
+          </Card>
+        </TabsContent>
```

Comment:
All of these TabsContent children should probably be components that you spin off into their own files. So you'd have a 
<CharacterNameTabsContent {...props} /> and so on. Each one would receive the activeTab state as a prop and only render if it matched its TabsContent value. You'll probably need to wrap all of these in a useForm and pass in the form's "controller" object that will make it a managed form. We mostly use `ReactHookForm` for form submissions: you'll need to create a form, react query mutation for submission, with that mutation hooked up to an api that'll match a corresponding API that you create in the backend for POST (creation) or PATCH (update) requests.